### PR TITLE
Update Chinese translate and fix Japanese and Russian some string error

### PR DESCRIPTION
**Pull request by dfr1238**
**When merged this pull request will:**
Update Chinese translate,and fix STR_ACRE_sys_list_PTT2Color_DisplayName in japanese and Russian

### CODE REVIEW

File: addons/sys_list/stringtable.xml [hunk 1]
```diff
@@ -35,73 +35,85 @@
...
             <English>Radio 1 (PTT 1) Transmit Color</English>
             <German>Farbe Sprechtaste 1</German>
             <Russian>Рация 1 (PTT 1) цвет передачи</Russian>
             <Japanese>無線機 1 (PTT 1) 送信カラー</Japanese>
+            <Chinese>無線電1 (隨按即說1)通訊顏色</Chinese>
         </Key>
         <Key ID="STR_ACRE_sys_list_PTT2Color_DisplayName">
             <English>Radio 2 (PTT 2) Transmit Color</English>
             <German>Farbe Sprechtaste 2</German>
             <Russian>Рация 2 (PTT 1) цвет передачи</Russian>
```
Comment (jonpas) [inline addons/sys_list/stringtable.xml]: This is also wrong, can you fix it as well?

Comment (dfr1238) [reply addons/sys_list/stringtable.xml]: ah,you mean which line?

Comment (dfr1238) [reply addons/sys_list/stringtable.xml]: oh,i saw it

**Review by jonpas — APPROVED** [review]

**Comment by jonpas** [comment]
Thank you!

**Pull request merged**